ziti-fabric/subcmd/inspect.go

```diff
@@ -49,36 +49,31 @@ func inspect(cmd *cobra.Command, args []string) {
 		if err != nil {
 			panic(err)
 		}
-		requestMsg := channel2.NewMessage(int32(mgmt_pb.ContentType_InspectRequestType), body)
-		waitCh, err := ch.SendAndWait(requestMsg)
+		requestMsg := channel.NewMessage(int32(mgmt_pb.ContentType_InspectRequestType), body)
+		responseMsg, err := requestMsg.WithTimeout(10 * time.Second).SendForReply(ch)
```

Comment:
so many other places in the other prs are "5 seconds" - this 10 second one stands out... please ensure it's correct and on purpose

Reply:
Yes. Because inspect may go out and talk to the routers, I gave it some extra time for the extra communication overhead